### PR TITLE
add vscode launch config for debugging inso

### PR DESCRIPTION
**Pull request by gatzjames**
Adds a vscode launch configuration for inso making it easy to debug from the editor.

inso_debug

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -61,7 +61,50 @@
       "dependsOn": [
         "Insomnia: Compile Renderer (Watch)",
         "Insomnia: Compile Main",
+        "Components: Compile (Watch)"
```
Comment (develohpanda) [inline .vscode/tasks.json]: Was this change intentional?

**Review by develohpanda — APPROVED** [review]
🚀 I left the second argument/prompt empty and inso prompted me as expected

image

File: .vscode/tasks.json (hunk 1 above)
Comment (gatzjames) [reply .vscode/tasks.json]: 😅 No forgot it cause I'm working on a launch config for when working with the components pkg 😁
